src/Printable.java

```diff
@@ -0,0 +1,5 @@
+public interface Printable {
+    default void print() {
+        System.out.println("Print content in console");
+    }
```

Comment:
👍

Reply:
Sorry, I have added class name and content printing.